src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -3758,6 +3809,67 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _showContextMenuAt(_toControlOrigin(cursorPos));
     }
 
+    double TermControl::CalculateQuickFixButtonWidth()
+    {
+        return GetPadding().Left;

```

Comment:
So, if the user has absolutely 0 padding, then we don't show them this button at all?